README.md

```diff
@@ -174,53 +174,44 @@ Known bug: the GUI has a cache, and deleting from the CLI may not become apparen
 
 #### `bot.watchAllChannelsForNewMessages(options)`
```

Comment:
heads up on the README here. I'm working on `thebearjew/documentation-js` which should build the method signatures, types, descriptions, and examples from the JSDOC comments in the source code.